ios/GoogleServices/Development/GoogleService-Info_1.plist

```diff
@@ -0,0 +1,36 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!DOCTYPE plist PUBLIC "-//Apple//DTD PLIST 1.0//EN" "http://www.apple.com/DTDs/PropertyList-1.0.dtd">
+<plist version="1.0">
+<dict>
+	<key>CLIENT_ID</key>
+	<string>185520625819-fcdc8edjttcc2u48uusvh2mcq0ah6ap5.apps.googleusercontent.com</string>
+	<key>REVERSED_CLIENT_ID</key>
+	<string>com.googleusercontent.apps.185520625819-fcdc8edjttcc2u48uusvh2mcq0ah6ap5</string>
+	<key>ANDROID_CLIENT_ID</key>
+	<string>185520625819-016knpp029t553482gvp224o10233rm0.apps.googleusercontent.com</string>
+	<key>API_KEY</key>
+	<string>AIzaSyDQAoRXRnewNwTcJV00Fe1TsvhEMF9hB74</string>
```

Comment:
Please make sure this file does not contain any sensitive information.

Reply:
Actually this file was added by mistake while merge with branch. I will remove this file.